odoo/service/server.py

```diff
@@ -1034,7 +1034,7 @@ def run(self):
                          len(odoo.modules.registry.Registry.registries))
             self.stop()
         except Exception:
-            _logger.exception("Worker (%s) Exception occured, exiting..." % self.pid)
+            _logger.exception("Worker (%s) Exception occurred, exiting..." % self.pid)
```

Comment:
```suggestion
            _logger.exception("Worker (%s) Exception occurred, exiting...", self.pid)
```

Reply:
@luzpaz not sure if intended but it looks like this code is back in your latest code

Reply:
sorry, i'm confused as to how. neverthless, fixed and